apps/client/components/ui/Heading.tsx

```diff
@@ -0,0 +1,8 @@
+interface Props {
+  readonly className?: string,
+  readonly title: string
+}
+
+export const Heading = ({ className, title }: Props) => (
+  <h2 className={`text-2xl text-white font-bold text-center ${className}`}>{title}</h2>
```

Comment:
Tu to samo co wcześniej. To h2 jak nie podasz className renderuje się z klasą 'undefined' 
clsx...